barretenberg/cpp/src/barretenberg/commitment_schemes/zeromorph/zeromorph.hpp

```diff
@@ -696,36 +696,16 @@ template <typename Curve> class ZeroMorphVerifier_ {
         Commitment C_zeta_Z;
         if constexpr (Curve::is_stdlib_type) {
             // Express operation as a batch_mul in order to use Goblinization if available
-            auto builder = rho.get_context();
+            auto builder = z_challenge.get_context();
             std::vector<FF> scalars = { FF(builder, 1), z_challenge };
             std::vector<Commitment> points = { C_zeta_x, C_Z_x };
             C_zeta_Z = Commitment::batch_mul(points, scalars);
         } else {
             C_zeta_Z = C_zeta_x + C_Z_x * z_challenge;
         }
 
-        // Receive proof commitment \pi
-        auto C_pi = transcript->template receive_from_prover<Commitment>("ZM:PI");
-
-        // Construct inputs and perform pairing check to verify claimed evaluation
```

Comment:
I thought this explanation of the meaning of the pairing inputs was helpful, no? Can we find a way to keep it?